vllm/engine/metrics.py

```diff
@@ -172,6 +176,8 @@ class Stats:
     num_running_sys: int
     num_waiting_sys: int
     num_swapped_sys: int
+    # Number of preemptions including both recompute and swap.
+    num_preemption_sys: int
```

Comment:
Sorry this is a huge nit, but `num_preemption` is now an `_iter` since we are calculating the number of `preemptions` during the laster `engine_step` iteration